src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -345,7 +349,9 @@ class DemuxFastqs
  @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
  var illuminaFileNames: Boolean = false,
  @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header.")
- val skipFailingReads: Boolean = false
+ val skipFailingReads: Boolean = false,
+ @arg(doc="Keep only reads identified as control if true, otherwise keep all reads. Control reads are determined from the comment in the FASTQ header.")
```

Comment:
```suggestion
 @arg(doc="Do not keep reads identified as control if true, otherwise keep all reads. Control reads are determined from the comment in the FASTQ header.")
```